### PR TITLE
3 32 overview close buttons

### PR DESCRIPTION
**Pull request by isantop**
Upstream GNOME has dropped custom assets for window close buttons in the overview. This PR allows Pop to follow suit. 

Upstream MR: https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/295

It also installs legacy versions of files to ensure those build correctly and have correct styling.

### CODE REVIEW

File: src/3.26/sass/widgets/_popovers-menus.scss [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  .popup-menu-arrow { } //defined globally in the TOP BAR
+  .popup-sub-menu {
+    background-color: $alt_base_color;
+    box-shadow: $shadow_0;
+  }
+
+  .popup-menu-content { padding: 8px 0; }
+  .popup-menu-item {
+    // min-height: $menuitem_size - 4px * 2;
```
Comment (mmstick) [inline src/3.26/sass/widgets/_popovers-menus.scss]: Should this comment be removed?

Comment (isantop) [reply src/3.26/sass/widgets/_popovers-menus.scss]: Yep! The GNOME Shell theme isn't currently linted and contains a lot of legacy code, so comments like this are pretty common. I'll get it removed.

**Comment by isantop** [comment]
@jackpot51 The build system is not properly detecting the correct version of GNOME Shell. It's building the GNOME 3.32 version on 19.04, 18.10, and 18.04.

**Comment by jackpot51** [comment]
@isantop the builds with my addition of the GNOME version debugging have the following output:

```
18.04: ** Installing the theme for GNOME version 3.28...
18.10: ** Installing the theme for GNOME version 3.30...
19.04: ** Installing the theme for GNOME version 3.32...
```

Is that correct?

**Comment by isantop** [comment]
Should be...

**Comment by isantop** [comment]
Aha, a file got moved somehow to the wrong location. That's what was causing it not to build the legacy close buttons for 3.28

**Pull request merged**